cmake/Helper.cmake

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 set(WASMEDGE_INTERPROCEDURAL_OPTIMIZATION OFF)
-if(CMAKE_BUILD_TYPE STREQUAL Release OR CMAKE_BUILD_TYPE STREQUAL RelWithDebInfo)
+if((CMAKE_BUILD_TYPE STREQUAL Release OR CMAKE_BUILD_TYPE STREQUAL RelWithDebInfo) AND NOT WASMEDGE_FORCE_DISABLE_LTO)
```

Comment:
You SHOULD NOT combine these two conditions in one if statement. Instead, you should have another if statement to check `WASMEDGE_DISABLE_LTO` and then set the `WASMEDGE_INTERPROCEDURAL_OPTIMIZATION` to `OFF`.